src/parser/retail/modules/items/WeaponEnhancementChecker.tsx

```diff
@@ -1,23 +1,30 @@
-import ITEMS from 'common/ITEMS';
-import { Enchant } from 'common/ITEMS/Item';
+import ITEMS from 'common/ITEMS/thewarwithin/enchants';
 import BaseWeaponEnhancementChecker from 'parser/shared/modules/items/WeaponEnhancementChecker';
 
 // Example logs with missing enhancement:
 // /report/tqbL2gBn8HvWAmrK/73-Mythic+Echo+of+Doragosa+-+Kill+(1:42)/Xepheris/standard/overview
 
+// const MIN_ENHANCEMENT_IDS = [
+//   ITEMS.ALGARI_MANA_OIL_R1.effectId,
+//   ITEMS.ALGARI_MANA_OIL_R2.effectId,
+//   ITEMS.IRONCLAW_WEIGHSTONE_R1.effectId,
+//   ITEMS.IRONCLAW_WEIGHSTONE_R2.effectId,
+//   ITEMS.IRONCLAW_WEIGHSTONE_R3.effectId,
+//   ITEMS.IRONCLAW_WHETSTONE_R1.effectId,
+//   ITEMS.IRONCLAW_WHETSTONE_R2.effectId,
+//   ITEMS.IRONCLAW_WHETSTONE_R3.effectId,
+//   ITEMS.OIL_OF_BELEDARS_GRACE_R1.effectId,
+//   ITEMS.OIL_OF_BELEDARS_GRACE_R2.effectId,
+//   ITEMS.OIL_OF_BELEDARS_GRACE_R3.effectId,
+//   ITEMS.OIL_OF_DEEP_TOXINS_R1.effectId,
+//   ITEMS.OIL_OF_DEEP_TOXINS_R2.effectId,
+//   ITEMS.OIL_OF_DEEP_TOXINS_R3.effectId,
+// ] as const satisfies number[];
+
 const MAX_ENHANCEMENT_IDS = [
-  // Dragonflight
-  ITEMS.PRIMAL_WHETSTONE_R3,
-  ITEMS.PRIMAL_WEIGHTSTONE_R3,
-  ITEMS.HOWLING_RUNE_R3,
-  ITEMS.HISSING_RUNE_R3,
-  ITEMS.BUZZING_RUNE_R3,
-  ITEMS.CHIRPING_RUNE_R3,
-  ITEMS.COMPLETELY_SAFE_ROCKETS_R3,
-  ITEMS.ENDLESS_STACK_OF_NEEDLES_R3,
-  ITEMS.FLAMETONGUE_WEAPON,
-  ITEMS.WINDFURY_WEAPON,
-].map((item) => (item as Enchant).effectId);
+  ITEMS.ALGARI_MANA_OIL_R3.effectId,
+  ITEMS.BUBBLING_WAX.effectId,
+] as const satisfies number[];
```

Comment:
there are specs currently that use whetstone/weighstone as the best temporary enchants (brewmaster is one) but they're a bajillion gold so very few actually do. probably worth including the r3 of each here.

Reply:
Oki good catch!